pkg/kubelet/volumes.go

```diff
@@ -106,11 +99,13 @@ func (vh *volumeHost) GetWriter() io.Writer {
 	return vh.kubelet.writer
 }
 
-// Returns the hostname of the host kubelet is running on
```

Comment:
Hmm.. I think it should only be removed if it is not true anymore. @pmorie can you clarify ?

Also a general rule we try to keep a PR or a commit focused on related changes so if someone uses `git blame` in the future they should be able to understand why a line was changed when looking at the overall commit.